server/controllers/upload.js

```diff
@@ -0,0 +1,23 @@
+const Profile = require('../models/Profile')
+const asyncHandler = require("express-async-handler");
+
+// @route POST /upload/profile
+// @desc Update user's profile photo
+// @access Public
+exports.updateProfilePhoto = asyncHandler(async (req, res, next) => {
+  const userId = req.user.id;
+  let profile;
+
+  if (!req.file) {
+    res.status(500).json({ error: "Couldn't upload photo" });
```

Comment:
This error could mean that file is not present in the request object. So it should be a client error, maybe a `400` (Bad Request)